config/initializers/new_framework_defaults_6_1.rb

```diff
@@ -7,7 +7,7 @@
 # Read the Guide for Upgrading Ruby on Rails for more info on each option.
 
 # Support for inversing belongs_to -> has_many Active Record associations.
-# Rails.application.config.active_record.has_many_inversing = true
+Rails.application.config.active_record.has_many_inversing = true
```

Comment:
I figure that one of our tests would catch this if it was a problem. Given that they didn't, I assume we're fine.